auth2client/src/components/Signin.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Signin extends Component {
+    state = {
+        username: "",
+        password: ""
+    };
+    
+
+    render() {
+        return (
+        <form onSubmit={this.handleSubmit}>
+
+            <div className="form-username">
+            <label htmlFor="username">Username</label>
+            <input
+                name="username"
+                value={this.state.username}
+                onChange={this.handleChange}
+                type="text"
+            />
+            </div>
+
+            <div>
+            <label htmlFor="password">Password</label>
+            <input
+                name="password"
+                value={this.state.password}
+                onChange={this.handleChange}
+                type="text"
+            />
+            </div>
+
+            <div>
+            <button type='submit'>Sign In</button>
+            </div>
+
+        </form>
+        );
+    }
+
+    handleChange = e => {
+        e.preventDefault();
+        this.setState({ [e.target.name] : e.target.value })
+    }
+
+    handleSubmit = e => {
+        e.preventDefault();
+        const creds = this.state;
+        const endpoint = 'http://localhost:4200/api/login';
+        axios.post(endpoint, creds)
+        .then(res => {
+            console.log('response data: ', res.data);
+            localStorage.setItem('jwt', res.data.token)
+        })
+        .catch(err => {
+            console.log('Error from Login', err)
```

Comment:
On line 58, the user will most likely not see the console.log statement. Think of a way to show the user that the login did now work. For example if the request fails you can set a boolean to state that then renders dynamic JSX whether the boolean is true or not